client/src/Home.js

```diff
@@ -1,21 +1,159 @@
+import { useLocation, useHistory } from "react-router-dom";
+
+import { BulgaLogo, BulgoFooter } from "./shared/logo/Avatars";
+import { Link } from "react-router-dom";
+
+import "./shared/style/Home.css";
+import PigCoin from "./shared/pics/undraw_Savings_re_eq4w (1).png";
+import LadyMoney from "./shared/pics/undraw_investment_data_yfvt.png";
+
+function Home({ setPath }) {
+  const location = useLocation();
+  const history = useHistory();
 
   useEffect(() => {
-    Api.budget.index().then((res) => setBudget(res.data));
+    setPath(location.pathname);
   }, []);
 
+  const handleLoginBtn = () => {
+    history.push("/login");
+  };
+  const handleSignUpBtn = () => {
+    history.push("/register");
+  };
+
   return (
-    <main className="container">
-      <header>
-        <h2>
-          <BudgetList budget={budget} />
-        </h2>
-      </header>
-    </main>
+    <div class="homeContainer">
+      <main className="topMain">
+        <div className="title">
+          <div className="titleHeading">Welcome to Bulga.</div>
+          <div className="titleSubHeading">
+            Simple budgeting to kickstart your savings
+          </div>
+        </div>
+        <header>
+          <nav className="navigation">
+            <div className="logo col col-md-6">
+              <Link to="/home">
+                <BulgaLogo />
+              </Link>
+            </div>
+            <ul className="col col-md-3">
+              <li>
+                <Link className="firstLink link" to="/home">
+                  Home
+                </Link>
+              </li>
+              <li>
+                <Link className="link" to="/resource">
+                  Resource
+                </Link>
+              </li>
+              <li>
+                <Link className="link" to="/FAQ">
+                  FAQ
+                </Link>
+              </li>
+            </ul>
+            <div className="register col col-md-3">
+              <button
+                onClick={handleLoginBtn}
+                id="login"
+                className="btn btn-primary"
+              >
+                Login
+              </button>
+              <button
+                onClick={handleSignUpBtn}
+                id="signUp"
+                className="btn btn-primary"
+              >
+                Sign up
+              </button>
+            </div>
+          </nav>
+        </header>
+      </main>
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        width="16"
+        height="16"
+        fill="currentColor"
+        class="bi bi-caret-down"
+        viewBox="0 0 16 16"
+        className="downIcon"
+      >
+        <path d="M3.204 5h9.592L8 10.481 3.204 5zm-.753.659 4.796 5.48a1 1 0 0 0 1.506 0l4.796-5.48c.566-.647.106-1.659-.753-1.659H3.204a1 1 0 0 0-.753 1.659z" />
+      </svg>
+      <main className="bottomMain">
+        <section className="topSection">
+          <div className="leftPart col-md-5">
```

Comment:
You're missing the needed container/row wrapper classes for the column classes to work here. You're making things more complicated by creating all of these custom classes instead of just using the bootstrap grid classes. I'll edit this myself to show what I mean, and hopefully you can follow from there...